dbinit/build.gradle

```diff
@@ -34,46 +32,22 @@ docker {
 
 task buildDockerImage(type: DockerBuildImage) {
 	inputDir = mkdir("build/docker")
-	tag = "${project.group}/yona-mariadb-liquibase-update:${project.ext.docker_tag}"
+	tags = [ "${dockerImageName}:${project.ext.docker_tag}" ]
 	doFirst {
 		copy {
 			from "src/main/liquibase"
-			into "$inputDir/changelogs"
+			into "$inputDir.asFile.orNull/changelogs"
 			exclude "liquibase-task.properties"
 		}
 		copy {
 			from "src/main/docker/Dockerfile"
 			from "src/main/docker/update.sh"
-			into "$inputDir"
+			into inputDir
 		}
 	}
 }
 
 task pushDockerImage(type: DockerPushImage, dependsOn: buildDockerImage) {
-	imageName = "${project.group}/yona-mariadb-liquibase-update"
+	imageName = dockerImageName
 	tag = project.ext.docker_tag
 }
-
-task pathingJar(type: Jar) {
```

Comment:
The pathingJar code was added by me because the dbinit project could not start because of too many dependencies in its class path. Is this now handled differently by the new Gradle version or Java version?

Reply:
It's worth trying. You might want to create a task ticket to see whether it's still necessary.

Reply:
I misunderstood this comment. I read it as a suggestion to remove it, but actually I removed it already.
It turns out we need to have this. It started failing on me today. I'll push it directly to the master.